src/lib/input/autosize.ts

```diff
@@ -15,27 +15,21 @@ import {Directive, ElementRef, Input, OnInit} from '@angular/core';
   },
 })
 export class MdTextareaAutosize implements OnInit {
-  /** Minimum number of rows for this textarea. */
+  /** @deprecated Use mdAutosizeMinRows */
```

Comment:
I'm not sure that having the `@deprecated` and the description on the same line is valid.